.github/workflows/ubuntu_x86_64.yml

```diff
@@ -0,0 +1,51 @@
+on: [pull_request]
+
+name: CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  test-zig-rust-wasm:
+    name: test zig, rust, wasm
+    runs-on: [self-hosted, i5-4690K]
+    timeout-minutes: 90
+    env:
+      RUSTC_WRAPPER: /home/small-ci-user/.cargo/bin/sccache
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          clean: "true"
+
+      - name: zig fmt check, zig tests
+        run: cd crates/compiler/builtins/bitcode && ./run-tests.sh
+
+      - name: zig wasm tests
+        run: cd crates/compiler/builtins/bitcode && ./run-wasm-tests.sh
+
+      - name: regular rust tests
+        run: cargo test --locked --release --features with_sound serde --workspace && sccache --show-stats
+
+      - name: test the dev and wasm backend # these tests require an explicit feature flag
+        run: cargo test --locked --release --package test_gen --no-default-features --features gen-dev && sccache --show-stats
```

Comment:
I believe gan-dev runs only the dev backend, and the wasm backend is tested with `--features gen-wasm` in the next step, so maybe this should be named "test the dev backend"

Reply:
Nice catch @ayazhafiz, I've made the change as suggested.